Improve the freeCodeCamp Podcast entry

QuincyLarson opened the pull request:
Before raising a PR and it getting merged, make sure you have completed the following:

### Things to do:

- [x] Add podcasts in appropriate categories in ascending order of podcast name
- [x] If there is no matching category, add a new category to the list in ascending order of category name, and add it to the table of contents in the same order
- [x] Add a brief description of the podcast
- [x] Add the frequency of episodes (Check the list for examples)
- [x] Add the runtime of episodes, giving an approximate range and an average duration.

Thanks for your contributions and being awesome :) Cheers.

rShetty commented on README.md:
```diff
@@ -579,10 +579,11 @@ More .NET Podcasts can be found on [The Sound of .NET](https://thesoundof.net/)
-  * **Description**:  A weekly talk that gives insights, people who started to code from non-technical background and successfully kickstarted thier carrers, as well as interviews with experts on how to get started in new hot topics in tech
-  * **Frequency** : Weekly Once
-  * **Runtime**: 10 - 30 mins, regularly ~15 mins
+* [The freeCodeCamp Podcast](https://podcast.freecodecamp.org/) ([iTunes](https://itunes.apple.com/us/podcast/the-freecodecamp-podcast/id1313660749) | [Spotify](https://open.spotify.com/show/7CpuEnbCLIXwI6LEcbBOYP) | [Stitcher](https://www.stitcher.com/podcast/freecodecamp-podcast))
```
This is not in sorted order, can you fix plz

QuincyLarson replied:
Sure - I've moved the entry so it's in alphabetic order.

Yanusih commented:
https://github.com/rShetty/awesome-podcasts/pull/201#issue-250469240

rShetty commented on README.md:
```diff
@@ -596,6 +591,12 @@ More .NET Podcasts can be found on [The Sound of .NET](https://thesoundof.net/)
   * **Frequency** : Once every week
   * **Runtime**: 45 - 70 mins, regularly ~55 mins
 
+* [The freeCodeCamp Podcast](https://podcast.freecodecamp.org/) ([iTunes](https://itunes.apple.com/us/podcast/the-freecodecamp-podcast/id1313660749) | [Spotify](https://open.spotify.com/show/7CpuEnbCLIXwI6LEcbBOYP) | [Stitcher](https://www.stitcher.com/podcast/freecodecamp-podcast))
```
It starts with `T` should probably move below

QuincyLarson replied:
The English language convention is to ignore the word "the" when alphabetically ordering things. People will be looking for this under the letter "f". Here's an example from Wikipedia: 
<img width="1172" alt="list_of_sovereign_states_-_wikipedia" src="https://user-images.githubusercontent.com/985197/52471566-2a07fc80-2bcc-11e9-8f7d-50e79b193fba.png">

rShetty replied:
I think we might have to reorder the others, if this is the case I think

QuincyLarson replied:
OK - do you want me to go through and reorder them all in accordance with this convention?

rShetty replied:
sure that should help. thanks

QuincyLarson commented:
@rshetty I've gone through and updated the order of all of the sections. Some of sections were already observing the "The" convention. I updated the remaining sections to also observe it. Thanks again for building and maintaining this comprehensive list of podcasts!

rShetty commented:
Thanks very much @QuincyLarson
